pkg/util/helpers.go

```diff
@@ -31,3 +32,23 @@ func GenerateConditionChangeEvent(t string, status types.ConditionStatus, reason
 		Message:   fmt.Sprintf("Node condition %s is now: %s, reason: %s", t, status, reason),
 	}
 }
+
+func GetDelayedStartTime(delay string) (time.Time, error) {
```

Comment:
I feel like this can be combined with lookback, just pass in both delay and lookback, and return starttime.

Reply:
Done.